### DISCUSSION TITLE
Binding Ikea Trådfri E1743 (FW 2.2.010) with Group

### DISCUSSION THREAD

**Discussion by ctankep23** [Koenkk/zigbee2mqtt#11952] [Devices]
I have a bunch of new Ikea switches that I'm psyched about adding to my HA with Z2MQTT. However I'm having real difficulties achieving it.

I'm trying to get the switch to control the living room lamp (5 bulbs - created a group in ZQMQTT) and it seems I can succesfully bind the s witch to the group, but nothing (physically) happens when I press either button on the switch. The log produces this:

`MQTT publish: topic 'zigbee2mqtt/Ikea Livingroom Ceiling Lamp Dimmer', payload '{"action":"brightness_move_up","action_rate":83,"battery":87,"linkquality":191,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Ikea Livingroom Ceiling Lamp Dimmer', payload '{"action":"","battery":87,"linkquality":191,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Ikea Livingroom Ceiling Lamp Dimmer/action', payload 'brightness_move_up'
Info MQTT publish: topic 'zigbee2mqtt/Ikea Livingroom Ceiling Lamp Dimmer', payload '{"action":"brightness_stop","battery":87,"linkquality":191,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Ikea Livingroom Ceiling Lamp Dimmer', payload '{"action":"","battery":87,"linkquality":191,"update":{"state":"idle"},"update_available":false}'
Info MQTT publish: topic 'zigbee2mqtt/Ikea Livingroom Ceiling Lamp Dimmer/action', payload 'brightness_stop'`

So the switch is at least doing its part, right? But the Group is not receiving the message..

On [the device page for E1743](https://www.zigbee2mqtt.io/devices/E1743.html) it states that an E1743 with FW lower than 2.3.75 supports binding to groups only. So why am I hitting a wall here, since the FW version is 2.2.010?

But I had to test, so I bound the switch to a single Ikea bulb and it just worked! I was unable to remove the binding to the group, and it worked anyway(?!).. So has there been some sort of update or am I missing something vital?

**Comment by r-vit** [comment]
Sometimes the firmware version reported by Z2M is not the real firmware of the device. You can check the real version by using the developer's console on your device page and issuing a request to the genBasic cluster and swBuildId attribute.
Mine says it's on 2.2.010 firmware while the request reports 2.3.079

**Reply by Diegocampy** [reply]
I'm not sure what I did, but if I get this 2.2.010 does it mean that actually that is the installed firmware? I wonder why it doesn't update or get updates.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/65871526/8236768a-43f9-49a2-9826-46c064a86245)
